Documentation/running-lkvm-stage1.md

```diff
@@ -1,8 +1,10 @@
 # Running rkt with an LKVM stage1
 
-rkt has experimental support for running with an [LKVM](https://kernel.googlesource.com/pub/scm/linux/kernel/git/will/kvmtool/+/master/README) stage1.
+rkt has experimental support for executing pods with an [LKVM](https://kernel.googlesource.com/pub/scm/linux/kernel/git/will/kvmtool/+/master/README) stage1.
 That is, rkt will start a virtual machine with full hypervisor isolation instead of creating a container using Linux cgroups and namespaces.
 
+This facility is labeled experimental to denote that the LKVM stage1 does not yet implement all of the default stage1's features and semantics. While the same app container can be executed above either stage1, it may require different configuration, especially for networking. However, several considered deployments above the LKVM stage1 are operational outside of CoreOS, and we encourage testing of the feature and welcome your contributions.
```

Comment:
IMO this should sound more like "can run using either...", but keep in mind that I'm far from being fluent in english language...